src/core/helper.js

```diff
@@ -16,6 +16,20 @@ const helper = {
   sum(array, accessor){
     return array.map(accessor)
       .reduce(((prev, current) => prev + current), 0);
+  },
+  functor(v) {
```

Comment:
Thanks for adding the check, but a helper function like this should be quite abstract and does not get too involved with how will be used. To be specific, this functor should be useful for other value not just nodeHeight or event limited to Number. This can be used for String, Object, etc. as well. In conclusion, I wouldn't add the check here. 

If you really want to add the check, it might be better to do this check in `getWaypoints()` and `layout` by checking the output value of  `nodeHeightFn()` regardless of the type of `options.nodeHeight`. That way you also validate if somebody accidentally set options.nodeHeight to non-Number constant.

The check can print the warning `nodeHeight does not return a Number, returning 10 instead of ${value}`.